.ado/publish.yml

```diff
@@ -10,6 +10,7 @@ trigger:
     include:
       - master
       - '*-stable'
+      - 'heyimchris/redbox'
```

Comment:
As I said, not planning to check this in, just need to trigger a publish that I can move to SDX-Platform and integrate into devmain

Reply:
why not? Having one-off publishes from temporary branches makes the publish not have a repeatable source of truth. Can we either add this 0.62-stable or create a new suffix or minor version and actually get this change committed to that branch? It seems like a small thing now that could prevent a major inconvenience if we need to take a another change off of 0.62-stable and don't want to lose this change?